corehq/elastic.py

```diff
@@ -139,10 +139,21 @@ class ESError(Exception):
     pass
 
 
-def run_query(index_name, q):
+def run_query(index_name, q, debug_host=None):
+    # the debug_host parameter allows you to query another env for testing purposes
+    if debug_host:
+        if not settings.DEBUG:
+            raise Exception("You can only specify an ES env in DEBUG mode")
+        es_host = getattr(settings, 'ELASTICSEARCH_HOST_{}'.format(debug_host.upper()))
+        es_instance = Elasticsearch([{'host': es_host,
+                                      'port': settings.ELASTICSEARCH_PORT}],
+                                    timeout=3, max_retries=0)
```

Comment:
There are two failure modes I expect here - you didn't define the appropriate localsetting, in which case you'll get a message about "settings has no attribute 'ELASTICSEARCH_HOST_PROD'", or you're not connected to the VPN, in which case I force a timeout after 3 seconds.  In both of these instances, I think the exceptions provide sufficient context that I don't need to check separately.